### PR TITLE
Feature/deeplink aggregation

### PR DESCRIPTION
**Pull request by pyby**
**API**
- API to get an up to date « parse Play url » JavaScript, including topics and event modules.
- API to send a failed parsing Play url.

- Up to 2500 reports maximum.
- Refresh the JavaScript cache every 5 minutes.

**Admin**
- Add failed parsing Play url reports tab.
- Don’t allow to add an empty update entry.
- Logout button.

### CODE REVIEW

File: src/main/java/ch/srgssr/playfff/service/UpdateService.java [hunk 1]
```diff
@@ -29,6 +29,7 @@ public void remove(String packageName, String version) {
         repository.removeByPackageNameAndVersion(packageName, version);
     }
 
+    @Transactional
```
Comment (amtins) [inline src/main/java/ch/srgssr/playfff/service/UpdateService.java]: Could you please check that ?

File: src/main/resources/parsePlayUrl.js [hunk 2]
```diff
@@ -0,0 +1,710 @@
...
+
+	var queryParams = {};
+	for (var queryItem of url.searchParams) {
+		queryParams[queryItem[0]] = queryItem[1];
+	}
+
+	return parseForPlayApp(url.hostname, url.pathname, queryParams, url.hash);
+}
+
+function parseForPlayApp(hostname, pathname, queryParams, anchor) {
```
Comment (amtins) [inline src/main/resources/parsePlayUrl.js]: Is this function intended to wrap the entire Internet ?

File: src/main/java/ch/srgssr/playfff/service/UpdateService.java (hunk 1 above)
Comment (pyby) [reply src/main/java/ch/srgssr/playfff/service/UpdateService.java]: Removed.

File: src/main/resources/parsePlayUrl.js (hunk 2 above)
Comment (pyby) [reply src/main/resources/parsePlayUrl.js]: Half the the internet is enough.
For sure, we should refactor it.

**Pull request merged**